src/TensorFlowNET.Keras/Engine/Model.Evaluate.cs

```diff
@@ -26,14 +26,17 @@ public partial class Model
         /// <param name="workers"></param>
         /// <param name="use_multiprocessing"></param>
         /// <param name="return_dict"></param>
+        /// <param name="is_val"></param>
         public Dictionary<string, float> evaluate(NDArray x, NDArray y,
             int batch_size = -1,
             int verbose = 1,
             int steps = -1,
             int max_queue_size = 10,
             int workers = 1,
             bool use_multiprocessing = false,
-            bool return_dict = false)
+            bool return_dict = false,
+            bool is_val = false
```

Comment:
Because when we call evaluate using validation_split in model.fit, we don't need to call the on_test_batch_end, which will interfere with our output on the screen. And when we call model.evaluate , we does need the on_test_batch_end. So I pass a parameter to distinguish between the two cases